lib/api.js

```diff
@@ -638,7 +638,12 @@ api.declare({
 });
 
 /**
- * List AMIs and their usage
+ * List AMIs and their usage by returning a list of objects in the form:
+ * {
+ *   region: string,
+ *   volumetype: string,
+ *   lastused: timestamp,
+ * }
```

Comment:
Thanks for this!  However, docs for API methods should be in the `description` field a few lines below.  That eventually makes its way to the documentation site so that others know how to call these API methods.  It's in markdown format.

Reply:
I just realized that there's this comment on lines 598 to 602 saying:
```
/*    NOTE:  ALL FOLLOWING METHODS ARE INTERNAL ONLY AND ARE NOT             */
/*           INTENDED FOR GENERAL USAGE.  AS SUCH THEY ARE ALL               */
/*           CONSIDERED TO BE EXPERIMENTAL, DO NOT HAVE ANY                  */
/*           SCHEMA DESCRIPTIONS AND ARE INTENDED TO BE CHANGED              */
/*           WITHOUT ANY NOTICE.                                             */
```
Should we still specify the schema for AMI and EBS usage then?

Also, where is the internal API documentation site?

Reply:
There isn't an internal API documentation site :)

Schema descriptions are formal, verifiable JSON-schema entries.  As the comment suggests, we don't need to add those.  But I think it's just neighborly to describe (in the description) what the results look like, to avoid misunderstandings and misinterpretation of the results.